README.md

```diff
@@ -104,6 +104,105 @@ Add the following dependency in your **module's build.gradle**.
       */
         Offix.enqueue(apolloClient, mutation, callback);
   ```
+  
+  
+ **Extend Background Offline Support of the library**
+ 
+ - Create a class `Worker` that extends `OffixWorker` of the library.
+ - Follow the below mentioned steps in the Worker's `doWork()` method.
+ 
+   - Get the list of mutations by calling `getListOfMutations()` function of the parent class.
+   - Check the network status.
+   - If the network is connected, then run a loop on list of mutations.
+   - On each mutation fetched from the list, create an object of Mutation<D,T,V> by passing the fetched mutation to the 
+     `getMutation(mutation)` function of library.
+   - You can use the above object while creating an instance of ApolloCall and make a call to the server.
+   - Make sure to delete the fetched mutation from the list in the onResponse() of your callback by calling              
+   `deleteMutation(mutation)`.
+   
+ - **In Kotlin**
+  
+  ```kotlin
+        /*
+         getListOfMutations() returns the list of mutations stored in database in the library.
+         It's present in the parent class, i.e OffixWorker.
+         */
+        val listOfMutations = getListOfMutations()
+
+        /*
+         Check again if the network connection is there or not.
+         */
+        if (Offline.isNetwork()) {
```

Comment:
The worker will be executed only if there's an active network.
Also, if you want to add a network check; do it inside the forEach loop.

Same applies for the sample app.